types/braces/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`.
If you have to disable it, you can disable the rule by disabling it in the source code.